common/static/common/js/spec/components/.gitignore

```diff
@@ -0,0 +1 @@
+*.js
```

Comment:
@andy-armstrong I have moved `feedback_spec.coffee` here, so during tests `coffee` is compiled into `.js` and we don't want to commit this js file so i am gitignoring it.